places/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
```

Comment:
you mentioned that you need it to be extendable, right here is where you can put the keyward param, here you can have {placeApi}
so in the index, you check for param and if it's not google, you can return `This place api is still in development`

Reply:
Hmm....ok